varcode/effects.py

```diff
@@ -115,6 +118,7 @@ class Intragenic(MutationEffect):
     apparently does happen sometimes, maybe some genes have two distinct sets
     of exons which are never simultaneously expressed?
     """
+    short_description = "intragenic"
```

Comment:
not always; coding effects usually give you the scientific notation of a variant. Example from https://github.com/hammerlab/varcode/blob/master/examples/varcode-quick_start.ipynb:

```
> myVariant.short_description
'chr7 g.140453136A>T'
```